podman cannot run as a limited user

Podman cannot run as a limited user because it lacks subuids / subgids.

galaxy_ng uses podman at least.
```
E           pulp_smash.exceptions.CalledProcessError: Command ('podman', 'pull', 'pulp/test-fixture-1:manifest_a') returned non-zero exit status 125.
E           
E           stdout: 
E           
E           stderr: time="2022-10-04T19:50:12Z" level=warning msg="\"/\" is not a shared mount, this could cause issues or missing mounts with rootless containers"
E           time="2022-10-04T19:50:12Z" level=error msg="cannot find UID/GID for user pulp: no subuid ranges found for user \"pulp\" in /etc/subuid - check rootless mode in man pages."
E           time="2022-10-04T19:50:12Z" level=warning msg="Using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding sub*ids if not using a network user"
E           Error: permission denied
```

--- a/CHANGES/304.bugfix
+++ b/CHANGES/304.bugfix
@@ -1,0 +1,1 @@
+Fix running podman as a limited user, like galaxy_ng requires.

--- a/s6_images/pulp_ci_centos/Containerfile
+++ b/s6_images/pulp_ci_centos/Containerfile
@@ -82,6 +82,7 @@ RUN sed 's|^#mount_program|mount_program|g' -i /etc/containers/storage.conf
 
 RUN groupadd -g 700 --system pulp
 RUN useradd -d /var/lib/pulp --system -u 700 -g pulp pulp
+RUN usermod --add-subuids 100000-165535 --add-subgids 100000-165535 pulp
 
 RUN mkdir -p /database \
              /etc/nginx/pulp \

